k6/README.md

```diff
@@ -0,0 +1,14 @@
+# K6 ReadMe
+
+## Introduction
+
+This readme is for how to install and us k6 for manual load testing
+
+## Installation
+
+The link for installation of k6 can be found [here](https://grafana.com/docs/k6/latest/get-started/installation/). Follow the instructions on the link for your installer of choice
+
+## Running locally
+
+1. Make sure local backend is running
+1. in a sepearate terminal run ```k6 run script.js``` (Where 'script' is the name of the script you want to run)
```

Comment:
Typo: `In a separate terminal`

Also, use single backticks for inline code, not triples.